crates/hulk/src/nao/hula.rs

```diff
@@ -363,7 +366,7 @@ impl From<types::Ear> for Ear {
     }
 }
 
-#[derive(Clone, Copy, Debug, Default)]
+#[derive(Clone, Copy, Debug, Default, PartialEq)]
```

Comment:
why is this needed?

Reply:
Mainly to compare the types in assertions in the tests.

Reply:
Ok, I was shortly thinking about a way to represent the fact, that it is only needed for testing. But never mind. I'm fine, if you have no further ideas.
(Feel free to resolve when read)